package.json

```diff
@@ -38,7 +39,7 @@
     "test": "mocha",
     "lint": "eslint \"**/*.js\" Jakefile",
     "coverage": "istanbul cover node_modules/mocha/bin/_mocha",
-    "doc": "rimraf out && jsdoc -c jsdoc.json lib/* docs/jsdoc/*",
```

Comment:
I know you're deleting this line -- but why would you install an entire module to `rm -rf` when you're already executing a shell command?